### ISSUE TITLE
[13] Project Overview: Disbale creation of new timesheet from Project Overview

### ISSUE THREAD

**Issue by sswapnesh** [odoo/odoo#52543]
Impacted versions: 13.0

Steps to reproduce: Project Overview and Create new Entry
![screenshot-2977716-13-0-8c4439 runbot31 odoo com-2020 06 05-23_41_22](https://user-images.githubusercontent.com/16624719/83909417-60389a80-a786-11ea-8dda-228089e1dfba.png)

Current behavior: Lines are not supposed to be created from this actions (and they wont be linked to Group)

Expected behavior: User should not be able to create new Lines from this actions

Video/Screenshot link (optional):

Ps: Looks like we are missing t-att-data-context="json.dumps({'create': False})" on actions

**Comment by pedrobaeza** [comment]
But is there any problem creating lines from there?

**Comment by sswapnesh** [comment]
> But is there any problem creating lines from there?

End user assumes "Creating lines from **Billed on Timesheets** will link it to the bills, **Non Billabel tasks** will not link those entries to any SO" which is misunderstanding.

**Comment by pedrobaeza** [comment]
Well, that's the same as creating a quotation from sales orders menu and don't confirm it, resting in the other menu, but let's see what Odoo says.

**Comment by sswapnesh** [comment]
Yes but that a part of record 's workflow/stages, Here there is no such thing.
